xmtp_mls/src/groups/device_sync.rs

```diff
@@ -404,10 +469,10 @@ where
 
         let _message_id = sync_group.prepare_message(&content_bytes, provider, {
             let request = request.clone();
-            move |_time_ns| PlaintextEnvelope {
+            move |now| PlaintextEnvelope {
                 content: Some(Content::V2(V2 {
                     message_type: Some(MessageType::DeviceSyncRequest(request)),
-                    idempotency_key: new_request_id(),
+                    idempotency_key: now.to_string(),
```

Comment:
what's the implication of using `now` vs `request_id`?

Reply:
It's just for idempotency. I switched to it to conform with how the main group send function.